README.md

```diff
@@ -1,2 +1,128 @@
-# raincatcher
-Feedhenry Raincatcher Core Repository
+# RainCatcher
+Feedhenry RainCatcher Core Repository
+
+## About RainCatcher
+
+## Developing
+
+RainCatcher supports and is tested on Node.JS@4 and should run on latest LTS.
```

Comment:
Add this as `prerequisites` or `required software chapter`
I will add that raincatcher supports latest 2 LTS versions. Currently 4.4.7 LTS and 6 LTS
Add mongo and redis without specifying versions